ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main__4.15-nightly.yaml

```diff
@@ -95,6 +99,29 @@ tests:
     - chain: redhat-chaos-hog-scenarios
     workflow: openshift-qe-installer-gcp
   timeout: 6h0m0s
+- as: krkn-hub-tests-aro-415
+  cron: 0 8 * * 1
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: azure-qe
+    env:
+      ARO_MASTER_VM_SIZE: Standard_D16s_v3
+      ARO_WORKER_VM_SIZE: Standard_D8s_v3
+      CHURN: "false"
+      GC: "false"
+      ITERATION_MULTIPLIER_ENV: "4"
+      PROFILE_TYPE: reporting
+      TELEMETRY_GROUP: prow-ocp-aro-4.15
+    test:
+    - ref: openshift-qe-cluster-density-v2
+    - chain: redhat-chaos-krkn-hub-etcd-tests
```

Comment:
We can replace the test cases with krkn-hub chain: https://github.com/openshift/release/blob/master/ci-operator/step-registry/redhat-chaos/krkn-hub-tests/redhat-chaos-krkn-hub-tests-chain.yaml + hog-scenarios. Thoughts?

Reply:
the prometheus chain doesnt work for aro: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/50795/rehearse-50795-periodic-ci-redhat-chaos-prow-scripts-main-4.15-nightly-krkn-hub-tests-aro-415/1818687176120995840